assets/main.scss

```diff
@@ -23,6 +23,8 @@ $grey-color-dark:  darken($grey-color, 25%);
 // Width of the content area
 $content-width:    800px;
 
+$on-smallPhone:    366px;
```

Comment:
With some widths I get a lot of overlap of the title and the menu, can we fix that by adjusting these numbers?

<img width="641" alt="screen shot 2017-04-17 at 21 40 49" src="https://cloud.githubusercontent.com/assets/891252/25111039/a2755bba-23b6-11e7-8967-df2a961ae938.png">